### ISSUE TITLE
Is there any way to use it with React 17?

### ISSUE THREAD

**Issue by EvilBozkurt** [AndreasFaust/react-ticker#41]
Is there any way to use it with React 17.0.2v ? Is it me because I'm a beginner or is it not possible to use it?

**Comment by blakejoy** [comment]
Just `npm install react-ticker --force`. Documentation mentions it has no dependencies besides React 16+.

**Comment by gk-git** [comment]
What you need to do is to change the peer dependencies of the react-ticker package to the following.

```
"peerDependencies": {
    "prop-types": "^15.8.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
```

I have just created a pull request, soon it will get merged to the main if there wasn't any problem with the changed configuration

**Comment by AndreasFaust** [comment]
Fixed with version 1.3.1.
